src/main/java/com/disraely/entities/BaseEntity.java

```diff
@@ -0,0 +1,35 @@
+package com.disraely.entities;
+
+import java.io.Serializable;
+
+public class BaseEntity implements Serializable {
```

Comment:
Now implement Serializable is not necessary, but I added this implementation because in the future that all entities can be serialized (convert to stream of bytes) is util for storage of the entity in files (as 'database') or transmitted over a network.

Reply:
hmmm But this is not happening now, so I think you can remove it right? I don't think you will need it in a close future

Reply:
Entendido 🙌